Api/ApplicationServiceInterface.php

```diff
@@ -10,7 +10,9 @@ interface ApplicationServiceInterface
     public function save(): array;
 
     /**
-     * @return void
+     * @param string $externalId
+     * @param string $status
+     * @return bool
```

Comment:
Widać niżej typy IMO zbędny doc, Chyba, że Magento zbiera z niego jakieś informacje 😁

Reply:
No właśnie zbiera i zbuntował się w czasie kompilacji, bo wyrzuciłem to w tej wersji i dostałem błędy, więc przywróciłem ten nadmiarowy komentarz. Resztę wywaliłem i nie generował błędów.

Reply:
xD